include/roboteam_ai/stp/invariants/WeHaveBallInvariant.h

```diff
@@ -5,13 +5,13 @@
 #ifndef RTT_WEHAVEBALLINVARIANT_H
 #define RTT_WEHAVEBALLINVARIANT_H
 
-#include "stp/invariants/BaseInvariant.h"
+#include "BaseInvariant.h"
```

Comment:
Why did you remove the path to this file?

Reply:
Because the file that is included is in the same folder